app/views/layouts/notifier.text.haml

```diff
@@ -1,4 +1,4 @@
 != yield
-!= t('notifier.email_sent_by_diaspora', :pod_name => pod_name) 
-!= t('notifier.to_change_your_notification_settings')
+!= t("notifier.email_sent_by_diaspora", pod_name: link_to(pod_name, AppConfig.pod_uri.site))
```

Comment:
`link_to` doesn't work in text emails. But most notifications contain a link to the post/comment including the domain, and the `edit_user_url` also includes the domain. So I think for users with text-mails this should be enough to find their pod. I think the `pod_name` like before is enough for text mails.

Reply:
Updated.